tests/common/common.py

```diff
@@ -173,25 +170,7 @@ def test_monitored_resource(self):
             self.assertTrue(found_resource.labels[label],
                 f'resource.labels[{label}] is not set')
 
-    def test_request_log(self):
-        if self.language not in ["nodejs"]:
-            return True
-        log_text = f"{inspect.currentframe().f_code.co_name}"
-        log_list = self.trigger_and_retrieve(log_text, "requestlog")
-        # Note: 2 logs are spawned, only one containing http_request prop.
-        log_entry = log_list[-1]
-        if log_entry.http_request is None:
-          log_entry = log_list[-2]
-        found_request = log_entry.http_request
-        if hasattr(self, 'request_props'):
-            for prop in self.request_props:
-                self.assertTrue(found_request[prop],
-                f'{prop} is not set')
```

Comment:
do we stop testing logging request metadata?

Reply:
I want to rethink how this is done. The goal of the environment tests is to ensure consistency across languages/environments, but this one is currently node-only. We should try to re-write it in a way that tests the same thing in all languages.

Note that for the node library specifically, request metadata is already better covered in [the system tests](https://github.com/googleapis/nodejs-logging/blob/864f18848c87a22c335b48e60a1d4505d3c9bd94/system-test/logging.ts#L620), so this one was a bit redundant